Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -1444,14 +1448,28 @@ def generate_webkit_secure_coding_impl(serialized_types, headers):
     result.append('')
     result.append('namespace WebKit {')
     result.append('')
+    result.append('static NSDictionary *dictionaryForWebKitSecureCodingType(id object)')
```

Comment:
We could make this return a RetainPtr<NSDictionary> so accumulatedDictionary doesn't need to add to the autorelease pool.

Reply:
Done